hivemind_core/protocol.py

```
@@ -316,6 +327,17 @@
             self.callbacks.on_invalid_protocol(client)
         except:
             LOG.exception("error on invalid_protocol callback")
+
+        try:  # let the binary protocol know about it
+            self.binary_data_protocol.callbacks.on_invalid_protocol(client)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)